test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_Java.cs

```diff
@@ -50,8 +50,8 @@ public async Task JavaProcess_Same_AfterHostRestart()
             Assert.True(javaProcessesAfter.Count() > 0);
             // Verify number of java processes before and after restart are the same.
             Assert.Equal(javaProcessesBefore.Count(), javaProcessesAfter.Count());
-            // Verify Java same java process is used after host restart
-            var result = javaProcessesBefore.Where(pId1 => !javaProcessesAfter.Any(pId2 => pId2 == pId1));
+            // Verify Java different java process is used after host restart
+            var result = javaProcessesBefore.Where(pId1 => javaProcessesAfter.Any(pId2 => pId2 == pId1));
```

Comment:
Is this expected? It's not clear how this change is affecting this scenario. (Also, should change the test name if it's expected).

Reply:
Yes, this expected with the change now - related to the comment above - We only want to start Language Worker Channels in "WebHost" Scope only when running in Azure - Placeholder Mode - Earlier code was unnecessarily complex with conditions per OS and also when running locally vs running in Azure

Reply:
Okay -- let's change this test name then so it matches the expectation.